dev/packages/alpha/redis-0.0.2/dataset/info/manifest.yml

```diff
@@ -1,5 +1,4 @@
 title: Redis info metrics
-release: ga
```

Comment:
Will currently work but we default to experimental which I assume we will hide in the Kibana UI in the future, so we should probably put this to `release: beta`?

Reply:
Yes, just fixed this :)